### PR TITLE
auto create img/ directory when insert image

### PR DESCRIPTION
**Pull request by stardiviner**

### CODE REVIEW

**Comment by Rahi374** [comment]
Thank you for the fix. However:

- If `./img/` doesn't exist, then the `(let ...)` expression will fail as well, therefore please move the directory autocreation to right before this expression.
- `Symbol's function definition is void: f-directory-p`, therefore please use `file-directory-p`

**Comment by stardiviner** [comment]
I updated my commit, and use `(unless (file-directory-p "./img") (make-directory "./img"))`.

**Comment by Rahi374** [comment]
You didn't fix either of the things I requested.
Additionally, please remove the `(unless ...)` expression.

File: org-notebook.el [hunk 1]
```diff
@@ -164,6 +164,8 @@
 						     ".png"
 						     )
 				))))
+    (or (f-directory-p "./img") (make-directory "./img"))
```
Comment (Rahi374) [inline org-notebook.el]: Change `f-directory-p` to `file-directory-p`

File: org-notebook.el [hunk 2]
```diff
@@ -164,6 +164,8 @@
...
+    (unless (file-directory-p "./img") (make-directory "./img"))
```
Comment (Rahi374) [inline org-notebook.el]: Remove this line.

File: org-notebook.el (hunk 1 above)
Comment (Rahi374) [reply org-notebook.el]: Also move this line to in between lines 140 and 141, otherwise this autocreate img directory is useless.

**Review by Rahi374 — CHANGES_REQUESTED** [review]

File: org-notebook.el (hunk 2 above)
Comment (stardiviner) [reply org-notebook.el]: Sorry about this, I forgot this. I will update commit.

**Comment by Rahi374** [comment]
Fixed in [https://github.com/Rahi374/org-notebook/commit/d90c4aeca2442161e6dd89de175561af85aace03](https://github.com/Rahi374/org-notebook/commit/d90c4aeca2442161e6dd89de175561af85aace03)

**Pull request closed without merging**